csrankings-j.csv

```diff
@@ -1188,6 +1191,7 @@ Jirí Kléma,Czech Technical University,https://ida.fel.cvut.cz/klema,zuwJZmEAAA
 Jiri Matas,Czech Technical University,http://cmp.felk.cvut.cz/~matas,EJCNY6QAAAAJ
 Jiwon Seo,Hanyang University,http://bdsl.hanyang.ac.kr/members/jiwonseo,O1WC6TAAAAAJ
 Jiyao An,Hunan University,http://csee.hnu.edu.cn/people/anjiyao,BeWWwfsAAAAJ
+Jizhong Zhao,Xi'an Jiaotong University,https://scholar.google.com/citations?user=CbGjM58AAAAJ,CbGjM58AAAAJ

```

Comment:
Google Scholar pages are not suitable as home pages. Please replace.

Reply:
Updated.